### DISCUSSION TITLE
A55DFCA4-205E-42E9-5C5A-6BF6684045FB

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#63] [General]
# M 31 (Andromeda Galaxy)

The Andromeda Galaxy (also known as Messier 31 or NGC 224) is an Sb-type barred spiral galaxy about 2.5 million light-years away in the constellation of the same name. Measuring around 220,000 light-years across, it is the largest galaxy of the Local Group. It is expected to collide with the Milky Way around 4-5 billion years into the future

https://celestia.mobi/addon?item=A55DFCA4-205E-42E9-5C5A-6BF6684045FB&lang=en

**Comment by AaronHaloXD** [comment]
AMAZINGGGG

**Comment by lalalalaallal271** [comment]
I dont like it since its a 2D model, i hope someone makes a 3D model for this.

**Reply by SevenSpheres** [reply]
There's a 3D model in the default program. Most galaxy addons like this are very old, from before Celestia supported galaxies by default.

**Comment by ta269** [comment]
Coooool

**Comment by Ksplover2343** [comment]
W

**Comment by hacker-ARSHAVIR-XR4** [comment]
When i installed it the when i both turned off galaxies and did not the andromeda galaxy was completely white. Does my phone not support the image used for the andromeda galaxy addon or is the addon broken?

**Reply by DaveBowman2001** [reply]
It is possible that your phone cannot support the image used on this add-on, but this is mostly caused by its pixel size not being equivalent to the powers of two (i.e. 512x512p, 2048x1024p, etc.). If that's the case, then this can be easily fixed by an update patch resizing the image file itself

**Reply by hacker-ARSHAVIR-XR4** [reply]
Thanks, i think its just the game itself still hasent patched this bug out on my device because im pretty sure this image used for the mod is on google on my device so my device supports it just celestia forgot to patch the bug on my device. Also my device is a iphone 7 plus and i think its iOS 14 if its a problem with my phone’s iOS then i will just update my phone to a higher iOS like iOS 16

**Reply by SevenSpheres** [reply]
The texture is 512x512 so the issue isn't its dimensions. Celestia doesn't get images from Google, the texture is included in the addon files. In any case I would recommend not using this addon, it doesn't add anything to the better-looking default Andromeda Galaxy (it was made way back in 2004 when there was no default Andromeda Galaxy).

**Reply by hacker-ARSHAVIR-XR4** [reply]
Can i just go adjust the texture sizes in my app files? If that works please tell me the exact dimensions i need to put in the file for it to work

**Reply by hacker-ARSHAVIR-XR4** [reply]
I cant believe jestr made it back in 2004 and hasn't updated it once yet 🤣

**Reply by SevenSpheres** [reply]
Please read my comment above. The texture size is already something that should work on all devices, you don't need to change it.

**Reply by thasnif369-sys** [reply]
It’s just like a white patch in mine😭

**Reply by DaveBowman2001** [reply]
> I cant believe jestr made it back in 2004 and hasn't updated it once yet 🤣

Jestr is no longer active years before Celestia even have an Android version itself. Please refrain from complaining against the addon developers themselves.

**Reply by developer1-vd** [reply]
It is actually due to there being No ttexture specified in the .dsc file.

**Reply by developer1-vd** [reply]
It is also due to `texture0` being set to `M31.PNG` and not `M31.png`.
